.github/workflows/cpp.yaml

```diff
@@ -140,6 +143,9 @@ jobs:
         - os: ubuntu-18.04
           build_type: Debug
           cxx: clang++-10
+        - os: ubuntu-18.04
+          build_type: Debug
+          cxx: clang++-12
```

Comment:
You are still allowing the clang-12 sanitizer build to run. Is that useful? If so that's fine, if not, go ahead and exclude it along with the debug build.

Reply:
Yes. A later version of the sanitizer should have some bugs fixed and improved error detection. Maybe shall we disable `clang-10` sanitizer instead and use just the `clang-12`?

Reply:
Let's let them overlap for a bit. In a week, lets consider disablin the clang-10 sanitizer.

Reply:
Ok, sure.